packages/dashboard/src/components/resourceExplorer/getCurrentAssets.ts

```diff
@@ -2,6 +2,17 @@ import { BranchReference } from '@iot-app-kit/source-iotsitewise';
 import { AssetSummary, AssetHierarchy } from '@aws-sdk/client-iotsitewise';
 import { HIERARCHY_ROOT_ID } from '.';
 import { AssetNode, BranchReferenceWithAssetIds, MaybeSiteWiseAssetTreeSessionInterface } from './types';
+import { ExtendedPanelAssetSummary, EitherAssetSummary } from '.';
+
+const rootAssetsHeaderItem: ExtendedPanelAssetSummary = {
+  isHeader: true,
+  name: 'Root Assets',
```

Comment:
Is this name displayed on the dashboard? If so can we include a section in messageOverrides?

Reply:
Added to messageOverrides